static/specification.html

```diff
@@ -144,595 +312,516 @@ <h2>
+            directive for request" on |request|.
+            </li>
+            <li>If the result of executing "Should fetch directive execute" on
+            |name|, `monetization-src` and |policy| is "No", return "Allowed".
+            </li>
+            <li>If the result of executing "Does response to request match
+            source list?" on |response|, |request|, this directive's value, and
+            |policy|, is "Does Not Match", return "Blocked".
+            </li>
+            <li>Return "Allowed".
+            </li>
+          </ol>
+        </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Security considerations
+      </h2>
+      <p>
+        It is RECOMMENDED that a user agent provide some UI or indicator that
+        allows the user to know when [=monetization=] is possible and/or when
+        [=monetization=] is occurring. Providing such a UI allows the users to
+        retain control of the monetization process by taking action (e.g., stop
+        or start monetization of a particular site if they wish to do so).
+      </p>
+      <p>
+        As [=payment end-points=] are generally provided as a service (e.g.,
+        Uphold), a XSS attack could inject a malicious [=payment end-points=]
+        into a page that uses the same service. To mitigate such an attack, it
+        RECOMMENDED that developers:
+      </p>
+      <ul>
+        <li>host a [=payment end-point=] on their own servers and request
+        [[SPSP]] details from the provider on the server as needed.
+        </li>
+        <li>Set the `monetization-src` CSP directive to restrict requests to
+        origins they control and trust to host [=payment end-points=].
+        </li>
+      </ul>
+    </section>
+    <section>
+      <h2>
+        Privacy considerations
+      </h2>
+      <p>
+        Web monetization is designed to be privacy-preserving: The user agent
+        does not send any data to the [=monetization provider=]. Instead, it
+        requests data from the [=monetization provider=] without ever revealing
+        the URL of the web page the user is currently browsing.
+      </p>
+      <p>
+        Further, the user agent gets the payment information from the [=payment
+        end-point=] to establish the [[STREAM]] connection. This also ensures
+        the [=monetization provider=] doesn't have access to a user's browsing
+        history or to the [=payment end-point=].
+      </p>
+    </section>
     <section id="relation-to-other-technologies" class="informative">
       <h2>
-        Relation to other technologies
+        Relation to Web Payments
       </h2>
       <p>
-        Unlike [[[payment-Request]]] and the [[[Payment-Handler]]], Web
+        Unlike [[[Payment-Request]]] and the [[[Payment-Handler]]], Web
         Monetization is intended for continuous payments rather than discrete
         payments. Additionally, it is designed to <em>not</em> require any user
```

Comment:
If tipping is WM, then WM is not just for continuous payments and also requires active user interaction. If tipping is not WM, we could still have WM via Open Payments, which would be recurring payments without user interaction.

Reply:
> If tipping is WM, then WM is not just for continuous payments and also requires active user interaction.

Right, the point we've been trying to make is that discreet and continuous payments are indistinguishable. They are just "payment" events that just happen to be of different monetary values (e.g, imagine a stream like this: .36c, .36c, .36c, $5, .36c, .36c, .36c,  $20... .36c,... and so on )